packages/maskbook/src/extension/background-script/StorageService.ts

```diff
@@ -48,7 +50,9 @@ class MutexStorage<T extends browser.storage.StorageValue> {
                     await browser.storage.local.set({ [key]: value })
                     callback(null)
                 } catch (e) {
-                    callback(e)
+                    if (e instanceof Error) {
+                        callback(e)
```

Comment:
This change actually changes the semantics and might break the code. Please always re-throw the error